content/blog/2020-04-27-moving-to-the-matrix/index.md

```diff
@@ -0,0 +1,185 @@
+  paying Slack to see the full history. But...
+- Slack is fairly expensive, and is charged per user. Depending on how we count
+  users, we have between approx. 150 users (those are the active ones) and over
+  a thousand (if you count all of them). Even if we got a discounted price (us
+  being an open source project and all), we'd still be looking at some serious
+  expenditure. That by itself is fine -- we all know that good software and
+  services, even if it's free software, is not free to build and we are willing
+  to pay for it. But if we're already paying, we might as well look at
+  alternatives, right?
+- Let's say we stuck with Slack and went to a paid plan with full history. We'd
+  still be locked into the Slack ecosystem. Slack is not open source, and stores
+  all chat history on proprietary serves owned by Slack themselves. As an open
+  source project, that should at least give us a pause to think if that's really
+  what we want.
+- The Slack clients are generally fine, and have definitively improved in recent
+  months. But overall, they're nothing special, and still can become CPU hogs.
+  Very few features in the actual clients are *that* important that we felt
+  compelled to stick with Slack.
+
+After considering all of this, we realized it's time to take a step back, and
+decide objectively how we can best organize the real-time communication (aka chat)
+service for GNU Radio.
+
+## How we picked a new service
+
+There were a few criteria we specifically looked out for:
+
+- There needed to be clients for Android and iOS, so users could easily
+  participate via their phones on the go.
+- Also, a web-based client was a must
+- Interaction with IRC, in particular Freenode, had to be possible. We still
+  consider `#gnuradio` on Freenode part of our GNU Radio infrastructure.
+- Cost should be reasonable. We weren't expecting a free service, but we also
+  don't want to squander GNU Radio funds.
+- There had to be an available service provider that would run the infrastructure
+  for us, so our GNU Radio leadership didn't have to manually run the servers.
+
+Some other criteria fell under the nice-to-have category:
+
+- All things equal, we would certainly prefer an open source solution to a
+  proprietary one.
+- An open standard for interoperability was considered important, if not
+  essential. This would allow easy interaction, e.g., between IRC and our chat
+  service.
+- Desktop clients and terminal-based clients were considered a great plus.
+- Ideally, this is a service that is used by other organizations as well.
+- GitHub integration and bells and whistles similar to that were considered a
+  plus as well.
+
+Using these criteria, we surveyed a bunch of options, including Mattermost,
+sticking with Slack, Discord, gitter, and of course Matrix.
+
+To test not only chat solutions, but also hosters, we rented a Mattermost and
+Matrix instance from a [Swiss hoster](https://unglei.ch) and tested them within
```

Comment:
As already noted in a channel it's https://ungleich.ch as the other domain is grabbed by a domain reseller